src/Frontend/Page/User.hs

```diff
@@ -262,11 +262,12 @@ instance FormPage EditUserProfile where
 editUserProfile :: ActionM m => FormPageHandler m EditUserProfile
 editUserProfile = FormPageHandler
     { _formGetPage   = EditUserProfile <$> currentUser
-    , _formProcessor = \up ->
+    , _formProcessor = \up -> do
+        uid <- currentUserId
         case up ^. profileAvatar of
-            Nothing -> throwError500 "upload FAILED: no file!"  -- FIXME: status code?
+            Nothing ->
+                update . SetUserProfileDesc uid $ up ^. profileDesc
```

Comment:
I guess this is what *should* work, so it's a good change, even though it doesn't make sense with the current behavior of df, df-lucid.